### PR TITLE
display academic code with institution name

### PR DESCRIPTION
**Pull request by kirti3192**
Displaying academic code with institution name while changing institution and requesting role of organiser.
It will help orginiser and training mngr to select the exact (correct) college.

### CODE REVIEW

File: events/forms.py [hunk 1]
```diff
@@ -89,13 +90,17 @@ def __init__(self, *args, **kwargs):
         if args:
             if 'state' in args[0]:
                 if args[0]['state'] and args[0]['state'] != '' and args[0]['state'] != 'None':
-                    choices = list(AcademicCenter.objects.filter(state_id = args[0]['state']).values_list('id', 'institution_name'))
+                    #choices = list(AcademicCenter.objects.filter(state_id = args[0]['state']).values_list('id', 'institution_name'))
+                    
+                    choices = list(AcademicCenter.objects.filter(state_id = args[0]['state']).annotate(combine=Concat('institution_name','academic_code')).values_list('id', 'combine'))
```
Comment (theskumar) [inline events/forms.py]: 1. The output of `value_list` is already a list so you might not want to have another `list()` function on top of it. 
2. You can just delete the comment above, keep a commented code just confuses everyone.

File: events/views.py [hunk 2]
```diff
@@ -2557,7 +2557,7 @@ def ajax_state_collage(request):
         tmp = '<option value = None> --------- </option>'
         if collages:
             for i in collages:
-                tmp +='<option value='+str(i.id)+'>'+i.institution_name+'</option>'
+                tmp +='<option value='+str(i.id)+'>'+i.institution_name+', '+i.academic_code+'</option>'
```
Comment (theskumar) [inline events/views.py]: Minor comment, this voilates PEP8 style guideline, there should be 1 space around operators.

**Comment by kirti3192** [comment]
@sanmugamk : Can you plz explain why we are filtering by different conditions in args and initial ?
In args using State_id,
[https://github.com/Spoken-tutorial/spoken-website/blob/production/events/forms.py#L92](https://github.com/Spoken-tutorial/spoken-website/blob/production/events/forms.py#L92)
and in initial using district_id,
[https://github.com/Spoken-tutorial/spoken-website/blob/production/events/forms.py#L98](https://github.com/Spoken-tutorial/spoken-website/blob/production/events/forms.py#L98)

**Comment by theskumar** [comment]
the changes look good. i'm going to do a bit of sanitation work and get this code in master.

**Comment by theskumar** [comment]
closed with #113

**Pull request closed without merging**